FishBun/src/main/res/layout/activity_detail_actiivy.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
An error occurs when entering the detail screen. Please check.

Reply:
Unfortunately I can not reproduce, see video here

https://www.icloud.com/iclouddrive/0F3QgjEbFTApV7GsPrU-PkzNw#fishbun

What error do you get, can you resolve it?

Reply:
In the video, only the PickerActivity is entered. If you click on a photo to enter DetailActivity, an error occurs.

Reply:
Do you mean when I am in the grid to select image and instead of selecting it click on the image to see it full screen? Still can not reproduce the error? What error do you get?

![Screenshot_1559796210](https://user-images.githubusercontent.com/5074917/59007405-e8602900-882e-11e9-8cbb-54aab2514c2a.png)

Reply:
I'll check again if it's only on my device. thank you